### ISSUE TITLE
Unclear meaning of configuration parameters

### ISSUE THREAD

**Issue by mion00** [basbruss/adaptive-cover#165]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/ludeeus/integration_blueprint/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Is your feature request related to a problem? Please describe.

Thank you for this great integration! I am eager to try it out with my outdoor (horizontal) awning, but I have some doubts in what the various parameters mean.
For instance windows height is described as:
> Height of the window when fully extended

Is it a typo? What does it mean for the window to be fully extended?

### Describe the solution you'd like

Does it perhaps mean the distance from the floor to the bottom part of the window?

### Describe alternatives you've considered

Maybe adding a simple diagram to the README might help avoid clear any doubts.

### Additional context

I have tried to draw a (very basic) sketch to explain my target setup. Are the labels indicated in the drawing correct?

![adaptive_cover drawio](https://github.com/basbruss/adaptive-cover/assets/4510647/a466db9a-994f-4661-a4c8-40bbf99ae898)

**Comment by bradkoerner** [comment]
I agree, a diagram would be immensely helpful. It would also be nice to have example use-cases for each of the parameters. It's hard to know why I'm setting parameters and what changing them will actually affect.

For example
|   |   |   |   |
| - | - | - | - |
| Minimal Elevation | None | 1-90 | Minimal elevation degree of the sun to be considered |
| Maximum Elevation | None | 1-90 | Maximum elevation degree of the sun to be considered |
|  |  |  |  |

Could instead read as 
|   |   |   |   |
| - | - | - | - |
| Minimal Elevation | None | 1-90 | Minimal elevation degree of the sun to be considered. Ex. There is a neighboring building that blocks the sun when it is below 20 degrees |
| Maximum Elevation | None | 1-90 | Maximum elevation degree of the sun to be considered Ex. There is fixed awning over the window that blocks the sun when it is higher than 80 degrees overhead |
|  |  |  |  |

I'm not even sure if my example is accurate to what that parameter controls, which is why I think diagrams and examples would be helpful.

**Comment by SirSheik** [comment]
Hello.
I am in the similar situation. But my roller blinds are vertical and I don't know if I'm setting the “Window Height” parameter correctly because I have window blinds that don't reach the floor and door blinds that start from the floor.
Could someone specify if the parameter “Window Height” refers to the size of the window (glass) or the distance from the bottom edge of the window to the floor?
Thank you very much
Best regards

**Comment by basbruss** [comment]
@SirSheik `window height` means the height of the window/glass.

**Comment by YayC** [comment]
100%, as someone new to this integration I'm confused by the parameter descriptions in the README. I suspect they were written by someone deeply familiar with the implementation so obvious things to them are not obvious to newcomers.

Would be great if someone who understands the parameters could improve the descriptions and add examples.

**Comment by homeontherocks** [comment]
> @SirSheik `window height` means the height of the window/glass.

Sorry to be pedantic here, but "height of the window/glass" could mean any of the following:
1.  Distance from the floor to the top of the window
2.  Distance from the floor to the bottom of the window
3.  Distance from the bottom of the window to the top of the window

Appreciate the clarification!

**Comment by SirSheik** [comment]
> 3\. Distance from the bottom of the window to the top of the window

Hi, 
Don't worry, I had the same doubt. I am no expert, but from the tests and after the summer, the distance from the window to the ground is not taken into account in the algorithm and it does not affect.
Regarding your question, it would be number 3.
_Distance from the bottom of the window to the top of the window_
I hope I have clarified your doubts.

**Comment by patrickp78** [comment]
3. Distance from the bottom of the window to the top of the window
In this case, the shadow distance is at the level of the bottom of the window. Need to estimate the right shadow distance if the working surface is different…

For me, I also could understand 
4. Distance from the working surface on which you want to calcul the shadow (desk on floor) to the top of the window
The height of the glazing is absolutely irrelevant.
This would make sense to calculate the shadow limit on a work surface, whether the window is full height or smaller, but I don't know if the calculations were done like that...

If this drawing is correct and can help…
![Image](https://github.com/user-attachments/assets/1284ccc5-d3ac-459a-9542-87a6b2dd1847)

**Comment by storm1er** [comment]
I'm starting to understand my issue. In my case, a have a table UNDER the awning. Meaning that when it's folded/closed, the sun at 90° elevation would directly slap chair and table with a nice and comfy burning ass temperature... Whatever settings I'm using, the awning is never fully deployed/opened when it should be 

Any idea how to fix this?
